spring/src/main/java/io/seata/spring/annotation/scannerexcluders/ScopeBeansScannerExcluder.java

```diff
@@ -0,0 +1,120 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.spring.annotation.scannerexcluders;
+
+import io.seata.common.loader.LoadLevel;
+import io.seata.spring.annotation.GlobalLock;
+import io.seata.spring.annotation.GlobalTransactionScanner;
+import io.seata.spring.annotation.GlobalTransactional;
+import io.seata.spring.annotation.ScannerExcluder;
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.aop.scope.ScopedProxyFactoryBean;
+import org.springframework.beans.factory.annotation.AnnotatedBeanDefinition;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.util.MultiValueMap;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Scope scanner excluder.
+ *
+ * @author wang.liang
+ */
+@LoadLevel(name = "ScopeBeans", order = 200)
+public class ScopeBeansScannerExcluder implements ScannerExcluder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScopeBeansScannerExcluder.class);
+    private static final Set<String> EXCLUDE_SCOPE_SET = Collections.synchronizedSet(new HashSet<>());
+
+    static {
+        EXCLUDE_SCOPE_SET.add("request");
+        EXCLUDE_SCOPE_SET.add("session");
+        EXCLUDE_SCOPE_SET.add("step");
+        EXCLUDE_SCOPE_SET.add("job");
```

Comment:
Please define constants for those names.

Reply:
done